creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -138,43 +191,43 @@ async function fetchExportFromSecondary({
     force_export: true
   }
```

Comment:
function diff:
- remove try-catch
- minor modification to "Validate export response" step